docs/releases.md

```diff
@@ -0,0 +1,39 @@
+# Releasing to NPM
```

Comment:
nit: "npm"

Reply:
I have a habit of capitalizing acronyms, especially ones that aren't word-ish.